### DISCUSSION TITLE
Flipped Parts

### DISCUSSION THREAD

**Discussion by AravindaDP** [usnistgov/ARIAC#250] [Q&A] [answered]
If my memory is correct, in previous year flipped parts challenge was limited to part type pump. (and may be battery also) The reasoning was that other parts geometry makes is difficult to flip using a single robot and robot malfunction could be part of the challenge.

Can we have similar constraint this year as well?

**Comment by zeidk** [comment, marked as answer]
Since ARIAC 2017, only one part type was requested to be flipped. We wanted to have more realistic scenarios this year by making all part types "flippable". We are also really interested to see the different approaches that competitors come up with to flip parts of different types. We understand that some parts are very small and 2 robots may be required to flip them. In this case, we will make sure to not have a robot malfunction challenge and a flipped part challenge in the same trial.
